SUMMARY.md

```diff
@@ -5,6 +5,7 @@
   - [Values](values.md)
 * [Our Virtual Office](onboarding.md)
   - [Onboarding](onboarding.md)
+  - [Coordination](coordination.md)
```

Comment:
```suggestion
  - [Coordination 🚧](coordination.md)
```

See https://github.com/hyphacoop/handbook/pull/29/files#r348088339

Reply:
I don't think we should put the emoji in the menu, it just makes more possibility to get out of sync.

Reply:
If it's a sync concern (which admittedly feels quite trivial), let's just put it only in menu where it is most useful for navigating.